### ISSUE TITLE
DL-3008 - The syntax is not recognized when using ENV var

### ISSUE THREAD

**Issue by SebastienFourreau** [hadolint/hadolint#329]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
Expected no error

### Actual behavior
Got an error

### Steps to reproduce the behavior
When i use an env var in my Dockerfile:
```bash
ENV PACKAGES_COMMON="python=2.7"
```
Error 3008 is trigger with:
```bash
apt-get install -yq --no-install-recommends $PACKAGES_COMMON
```

While this direct syntax works:
```bash
apt-get install -yq --no-install-recommends python=2.7
```

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:Haskell Dockerfile Linter v1.17.0-0-g56fd3e1

```bash
/dev/stdin:17 DL3008 Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`
```

**Dockerfile** (if relevant)

```Dockerfile
ENV DEBIAN_FRONTEND="noninteractive"

ENV PACKAGES_COMMON="python=2.7"
RUN echo "*** Installing common packages: $PACKAGES_COMMON ***"  && \
    apt-get -y update && \
    apt-get install -yq --no-install-recommends $PACKAGES_COMMON && \
```

**Additional environment details (OS, stack version, etc.)**

**Comment by lorenzo** [comment]
It is currently not, and we don't have a good way of interpolating variables in a reliable way, since some come from `ARG` and are not defined with an actual useful value before building the dockerfile.

My suggestion for the time being is to ignore the rule

**Comment by SebastienFourreau** [comment]
I understand :)
Thank you for the response and responsiveness.

Have a nice day
